### ISSUE TITLE
Webengine backend: Textfields ignore umlauts

### ISSUE THREAD

**Issue by mfrischknecht** [qutebrowser/qutebrowser#2312]
I know the `webengine` backend is still experimental, but I thought I might as well open an issue for this anyway, just so it is being tracked.

Using the `webengine` backend, umlaut combinations for the US-INTL keyboard layout (i.e. `"`+`a` = `ä`) work fine in qutebrowser's colon-menu, but are ignored if I type them in text fields or text areas (they don't produce a character at all. The default backend doesn't have this issue.

---

xkbmap:

```
> setxkbmap -print
xkb_keymap {
        xkb_keycodes  { include "evdev+aliases(qwerty)" };
        xkb_types     { include "complete"      };
        xkb_compat    { include "complete"      };
        xkb_symbols   { include "pc+us+us(intl):2+inet(evdev)+capslock(grouplock)"      };
        xkb_geometry  { include "pc(pc105)"     };
};
```

---

Version Info:

```
qutebrowser v0.9.1
Git commit: 

CPython: 3.6.0
Qt: 5.8.0
PyQt: 5.7.1

sip: 4.19
colorama: no
pypeg2: 2.15
jinja2: 2.9.5
pygments: 2.2.0
yaml: 3.12
cssutils: no
typing: yes
PyQt5.QtWebEngineWidgets: yes
pdf.js: no
Webkit: 538.1
SSL: OpenSSL 1.0.2k  26 Jan 2017

Style: QFusionStyle
Platform: Linux-4.9.6-1-ARCH-x86_64-with-arch-Arch-Linux, 64bit
Frozen: False
Imported from /usr/lib/python3.6/site-packages/qutebrowser
OS Version: 

--- /etc/arch-release ---
Arch Linux release

--- /etc/os-release ---
NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
ID_LIKE=archlinux

--- /etc/lsb-release ---
LSB_VERSION=1.4
DISTRIB_ID=Arch
DISTRIB_RELEASE=rolling
DISTRIB_DESCRIPTION="Arch Linux"

Paths:
config: /home/manuel/.config/qutebrowser
data: /home/manuel/.local/share/qutebrowser
system_data: /usr/share/qutebrowser
cache: /home/manuel/.cache/qutebrowser
download: /home/manuel/Downloads
runtime: /run/user/1000/qutebrowser
```

**Comment by The-Compiler** [comment]
Can you try if they work in QupZilla? Do they work after a restart, or are they always broken? I wonder if this is yet another case of [QTBUG-58635](https://bugreports.qt.io/browse/QTBUG-58635).

**Comment by mfrischknecht** [comment]
Thanks for your swift response :)

Yes, QupZilla shows the same behavior for me. It seems to stay consistent though; multiple restarts of qutebrowser/QupZilla and two reboots, and the keys never worked once.

On another note: I noticed, that I can't run two separate qutebrowser instances with different backends (it automatically attaches to the instance that's already running). Is this due to technical reasons (Firefox behaves the same way)? I've had a few occasions where two separate instances would be rather handy, and always had to resort to using another browser (coming from FF).

Also, I seem to have found a way to produce a crash: Closing one window after having launched qutebrowser a second time. After relaunching, I get a popup stating that the crash reporter was disabled since `QtWebKit` is not maintained any more; although I get the same crash with the `WebEngine` backend. Should I open another issue for this?

**Comment by The-Compiler** [comment]
> Yes, QupZilla shows the same behavior for me. It seems to stay consistent though; I tried multiple restarts of qutebrowser/QupZilla and two reboots, and the keys never worked once.

This is a QtWebEngine bug then - nothing I can do about that unfortunately, so I'm closing this.

> On another note: I noticed that I can't run two separate qutebrowser instances with different backends (it automatically attaches to the instance that's already running). Is this due to technical reasons (Firefox behaves the same way)? I've had a few occasions where two separate instances would be rather handy, and always had to resort to using another browser (coming from FF).

It is - however you can two instances with a different `--basedir`, or with `--temp-basedir` given. This duplicates all data (config, cookies, etc).

> Also, I seem to have found a way to produce a crash: Closing one window after having launched qutebrowser a second time.

Fixed in git master (it appeared with Qt 5.8), I just haven't released v0.10.0 yet.

> I get a popup stating that the crash reporter was disabled since QtWebKit is not maintained any more; although I get the same crash with the WebEngine backend. 

It's also reenabled in master with QtWebEngine
